migrations/V117__Add_ TRB_Request_Attendee_Table.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TYPE person_role AS ENUM (
+    'PRODUCT_OWNER',
+    'SYSTEM_OWNER',
+    'SYSTEM_MAINTAINER',
+    'CONTRACT_OFFICE_RSREPRESENTATIVE',
+    'CLOUD_NAVIGATOR',
+    'PRIVACY_ADVISOR',
+    'CRA',
+    'OTHER',
+    'UNKNOWN'
+);
+
+CREATE TABLE trb_request_attendees (
+    id UUID PRIMARY KEY NOT NULL,
+    trb_request_id uuid NOT NULL REFERENCES trb_request(id),
+    eua_user_id TEXT NOT NULL CHECK (eua_user_id ~ '^[A-Z0-9]{4}$'),
+    component TEXT NOT NULL,
+    role person_role NOT NULL DEFAULT 'UNKNOWN',
+    created_by TEXT NOT NULL CHECK (created_by ~ '^[A-Z0-9]{4}$'),
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    modified_by TEXT CHECK (modified_by ~ '^[A-Z0-9]{4}$'),
+    modified_at TIMESTAMP WITH TIME ZONE
+);
```

Comment:
Do you think it could be worthwhile to add a constraint here? Right now you can add the same EUAID attendee to the model many times. I don't imagine you should be allowed to, do you?

Could do something like
```sql
ALTER TABLE trb_request_attendees
ADD CONSTRAINT unique_attendee_per_request UNIQUE (trb_request_id, eua_user_id);
```
![image](https://user-images.githubusercontent.com/99294987/191803098-ebae9d48-0200-4f2d-ae7c-3c06f0bfaa4b.png)

Reply:
Ah yeah, I wrote something similar for intake contacts. Didn't think of it here.